src/konfo_backend/search/response.clj

```diff
@@ -11,6 +10,11 @@
   [response]
   (map (fn [x] (-> (:_source x) (assoc :_score (:_score x)))) (get-in response [:hits :hits])))
 
+(defn- autocomplete-hits
+  [response lng]
+  (map (fn [x] (let [res (:_source x)](assoc {} :label (get-in res [:nimi (keyword lng)]) :id (:oid res))))
```

Comment:
ehkä vähän enteriä kaipaisin tähän.

Reply:
Associn sijaan voinee myös vaan määritellä {:label (get-in jne.) :id (:oid res)}